.buildkite/pipeline.yml

```diff
@@ -36,6 +36,14 @@ steps:
           image: "python:3.6"
           propagate-environment: true
 
+  - command:
+      - "python -m pip install tox"
+      - "tox -e check-sampleconfig"
+    label: "\U0001F9F9 check-sample-config"
+    plugins:
+      - docker#v3.0.1:
+          image: "python:3.6"
+
```

Comment:
This seemed to take over 1 minute, so I'd be tempted to move after the wait

Reply:
yeah, it's downloading and installing all of the python deps. It seems a shame that (a) we can't do better caching here, and (b) the virtualenv isn't shared between the builds.